code/__HELPERS/roundend.dm

```diff
@@ -192,7 +192,9 @@
 	//Set news report and mode result
 	mode.set_round_result()
 
-	send2irc("Server", "Round just ended.")
+	var/survival_rate = GLOB.joined_player_list.len ? "PERCENT(popcount[POPCOUNT_SURVIVORS]/GLOB.joined_player_list.len)%" : "there's literally no players"
```

Comment:
```suggestion
	var/survival_rate = GLOB.joined_player_list.len ? "PERCENT(popcount[POPCOUNT_SURVIVORS]/GLOB.joined_player_list.len)%" : "there's literally no player"
```
Rogue `s` on the last word. Otherwise all good.